### PR TITLE
Update Engineering Manager ladder

### PR DESCRIPTION
**Pull request by SamRozen**

### CODE REVIEW

File: careers/ladder.md [hunk 1]
```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
...
+          <li>Knows how to run 1:1s effectively; facilitates constructive development conversations and reviews; and participates in Engineering Calibration.</li>
+          <li>Ensures engineers understand how the work they are doing ladders up to OKRs & wider company goals.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Ensures that their reports consistently deliver high quality work.</li>
+          <li>Active contributor in a given product / platform team.</li>
+          <li>Proactively monitors team health and delivery.</li>
+          <li>Identifies team dysfunctions and make them visible in forum like team retrospective or the manager forum when necessary.</li>
```
Comment (admbtlr) [inline careers/ladder.md:641]:
```suggestion
          <li>Identifies team dysfunctions and, where necessary, makes them visible in contexts such as team retrospectives or manager forums.</li>
```

File: careers/ladder.md [hunk 2]
```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
...
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Builds strong ties to the rest of the engineering manager community and activity participates in manager-wide forums.</li>
+          <li>Knows how to get high quality feedback from PMs, TLs and other members of their report's teams.</li>
```
Comment (admbtlr) [inline careers/ladder.md:647]:
```suggestion
          <li>Knows how to get high quality feedback from PMs, TLs and other members of their reports' teams.</li>
```

File: careers/ladder.md [hunk 3]
```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
...
+          <li>Identifies team staffing needs.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Will usually have 2-4 direct reports.</li>
+          <li>Provides timely, actionable feedback.</li>
+          <li>Get support from their manager to establish the right growth path for their reports.</li>
```
Comment (admbtlr) [inline careers/ladder.md:655]:
```suggestion
          <li>Enlists support from their manager to establish the right growth path for their reports.</li>
```

File: careers/ladder.md [hunk 4]
```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
...
+          <li>Can own a hiring pipeline and drive improvements to successfully fill open roles with high quality candidates without compromising on diversity within a reasonable timeline.</li>
+          <li>Effectively coaches senior engineers in a way that lines up with team goals.</li>
+          <li>Knows how to help engineers identify their superpowers and increase their impact by leaning into their strengths.</li>
+          <li>Addresses performance issues in a timely manner and with empathy.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Works closely with the team to meet the sprint goals; significantly accelerates the delivery and increases the impact of the team throughout the quarter.</li>
+          <li>Addresses effectively team health issues in their own team and their reports' team when need be.</li>
```
Comment (admbtlr) [inline careers/ladder.md:674]:
```suggestion
          <li>Effectively addresses team health issues both in their own and their reports' teams when need be.</li>
```

File: careers/ladder.md [hunk 5]
```diff
@@ -582,89 +582,212 @@ of their team considerably more than the direct impact of the code they write.
...
+          <li>Is accountable for long term technical roadmap to support the business needs and unlock growth.</li>
+        </ul>
+      </td>
+      <td class="border-top">
+        <ul>
+          <li>Drives PDDE changes in close partnership with PDDE Leadership team.</li>
+          <li>Actively participates in cross-org leadership activities.</li>
+          <li>Leverages network across the company to develop strong empathy for the various business needs.</li>
+          <li>Can give effective feedback on and influence company strategy.</li>
+          <li>Regularly delegates and increases own impact by helping others to achieve more.</li>
```
Comment (admbtlr) [inline careers/ladder.md:753]: I would expect this to be true from M3 up

File: careers/ladder.md [hunk 6]
```diff
@@ -563,15 +563,15 @@ See
...
-work. As such, an Engineering Manager has direct reports and their performance is evaluated by the scale and impact
-of their team considerably more than the direct impact of the code they write.
+A manager’s role at Artsy balances people management responsibilities with team delivery responsibilities. As a
+people manager, they are responsible for hiring, retaining, growing and managing the performance of their reports.
+Additionally, they're expected to significantly impact teams' delivery by providing a strong technical contribution
+and / or contribute to delivery management. As an engineering manager grows at Artsy, they scope of impact will
+increase from one team to multiple teams and from managing ICs to managing ICs and other EMs.
 
-## People Management vs Technical Leadership
+### People Management vs Technical Leadership
```
[the remark this answers is not in the document]
Comment (SamRozen) [reply careers/ladder.md:574]: It's mostly historical, when the TL role was introduced, I believe there was a lot of emphasis on clarifying the TL role as not being a title nor a promotion. Just an additional way to take on more / different responsibilities. I believe that at that point the team lead was also managing the people in their team. Happy to remove that section altogether if we believe it's not necessary anymore. Curious to get @starsirius or @sweir27's take on this section given that you have the most historical context.

**Comment by SamRozen** [comment]
Thanks a lot @admbtlr with your help to get the right wordings 🙏

File: careers/ladder.md [hunk 7]
```diff
@@ -563,15 +563,15 @@ See
...
+## Engineering Management
 
-A manager’s role at Artsy balances few individual contributor responsibilities with more people management
-responsibilities. As an engineering manager grows at Artsy, any code they write is expected to shift from
-delivering direct value on project progress to having a multiplying effect by enabling their team to do better
-work. As such, an Engineering Manager has direct reports and their performance is evaluated by the scale and impact
-of their team considerably more than the direct impact of the code they write.
+A manager’s role at Artsy balances people management responsibilities with team delivery responsibilities. As a
+people manager, they are responsible for hiring, retaining, growing and managing the performance of their reports.
+Additionally, they're expected to significantly impact teams' delivery by providing a strong technical contribution
```
Comment (joeyAghion) [inline careers/ladder.md:570]: Suggested rewording to remove repeated or ambiguous words:

> Additionally, they're expected to significantly impact teams' success through their own technical contribution and/or delivery management.

Personally, I'd also love to clarify what's meant by delivery management. I think we mean many things like team process, tools, release strategy, external coordination and communication.... Is there a concise way to capture this?

File: careers/ladder.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (sweir27) [reply careers/ladder.md:574]: Given we already have a section above on `Technical Leads`, I'd support removing this section.

(I do believe it was intended to clarify the state of affairs once we switched from having TL-managers as a single role to multiple).

[the remark this answers is not in the document]
Comment (SamRozen) [reply careers/ladder.md]: alright I removed it.

**Comment by SamRozen** [comment]
Note: This PR should be squashed before being merged.

File: careers/ladder.md (hunk 7 above)
Comment (SamRozen) [reply careers/ladder.md]: good call, I updated it.

File: careers/ladder.md (hunk 5 above)
Comment (SamRozen) [reply careers/ladder.md:741]: Good observation thanks! What I actually mean here is that at M3 I'd expect this to happen at the team level, and at M5, I'd expect that to happen beyond their team and across a large chunk of the business / company. Let me think of a better way to phrase this / clarify.

File: careers/ladder.md (hunk 7 above)
Comment (SamRozen) [reply careers/ladder.md]: Couldn't find a concise way to definer delivery management, so I went for the verbose way instead. It's still better than not defined at all.

**Review by joeyAghion — APPROVED** [review]

**Review by admbtlr — APPROVED** [review]
Everything looks great now, thanks @SamRozen !

**Pull request merged**